From 7696872e134d72af9cff1cefab10114ad9e1d2c4 Mon Sep 17 00:00:00 2001
From: Zongchun Yu <zongchun.yu@nxp.com>
Date: Wed, 5 Sep 2018 11:09:22 +0800
Subject: [PATCH] plat/nxp: fix tool build issue

Signed-off-by: Zongchun Yu <zongchun.yu@nxp.com>
---
 plat/nxp/tools/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plat/nxp/tools/Makefile b/plat/nxp/tools/Makefile
index e648fd1..43d3b56 100644
--- a/plat/nxp/tools/Makefile
+++ b/plat/nxp/tools/Makefile
@@ -32,6 +32,7 @@ endif
 INCLUDE_PATHS :=
 
 HOSTCC ?= gcc
+CC = gcc
 
 .PHONY: all clean distclean
 
-- 
2.7.4

